addons/hr/models/res_users.py

```diff
@@ -361,3 +362,45 @@ def action_related_contact(self):
             'res_model': 'res.partner',
             'view_mode': 'form',
         }
+
+    def _get_store_avatar_card_employee_prefetch_fields(self):
+        if not self.env.user._is_internal():
+            return []
+        base_fields = [
+            "work_email",
+            "work_phone",
+            "work_location_id",
+            "department_id",
+            "company_id",
+        ]
+        if self.env.user.has_group("hr.group_hr_user"):
+            # job_title is not a field of hr.employee.public, but it is a field of hr.employee
+            base_fields.extend(
+                [
+                    "job_title",
+                ]
+            )
+        return base_fields
+
+    def _get_store_avatar_card_employee_fields(self):
```

Comment:
`_get_store_avatar_card_employee_prefetch_fields` should be replaced by using `_get_store_avatar_card_employee_fields`

You can just iterate the result, and get the real field name with:
``` 
        def get_field_name(field_description):
            if isinstance(field_description, Store.Attr):
                return field_description.field_name
            return field_description
```
already present in channel file too